collections/ansible_collections/massopencloud/esi/roles/l2/tasks/set_networks_for_node.yaml

```diff
@@ -0,0 +1,45 @@
+- name: Grab current network configuration for node
+  block:
```

Comment:
Same comment re: `block:` here...

Reply:
Oh, ha, I wasn't sure - I think I picked this up from the floating IP playbooks!